Source/Dafny/Dafny.atg

```diff
@@ -3540,7 +3562,8 @@ LetExprWithLHS<out Expression e, bool allowSemi, bool allowLambda, bool allowBit
                                          }
                                        }
                                     .)
-    | ":-"                          (. isLetOrFail = true; .)
+    | {LambdaSpec<ref reads, ref req>}
```

Comment:
I think the rest of the file puts a space between the curly braces and what follows or comes before.